docs/pages/web/textfield.js

```diff
@@ -689,22 +693,93 @@ function Example(props) {
 - "previous": taking the user to the previous field that will accept text
 - "search": taking the user to the results of searching for the text they have typed
 - "send": delivering the text to its target
+
+The \`inputMode\` prop presents to the user a more accurate action label for the enter key on virtual keyboards. These are the values for each use case:
+
+- "none": No virtual keyboard. For when the page implements its own keyboard input control, for example DatePicker displays the calendar picker.
+- "text": Standard input keyboard for the user's current locale.
+- "decimal": Fractional numeric input keyboard containing the digits and decimal separator for the user's locale (typically "." or ",").
+- "numeric": Numeric input keyboard, but only requires the digits 0–9.
+
+Use \`type\` when TextField needs to capture phone numbers, emails or URLs.
```

Comment:
Would including those as `inputMode`s also be helpful? Or should we add those automatically for their respective `type`s?

Reply:
![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/10593890/229649149-7d7e7e2f-1572-4612-a69d-16ca5f281a2b.png)

Reply:
Got it, good call 👍